lxd/cluster/config/config.go

```diff
@@ -268,6 +268,11 @@ func (c *Config) ClusterHealingThreshold() time.Duration {
 	return healingThreshold
 }
 
+// OpenFGA returns all OpenFGA settings need to interact with an OpenFGA server.
+func (c *Config) OpenFGA() (string, string, string) {
```

Comment:
I'm not usually a fan of named returns but here it might be useful to know the order of the returns since they are all of the same type.